lib/get-wrapper.js

```diff
@@ -106,13 +106,15 @@ function getWrapper(ast, getReturn) {
         returnStmt = getReturnStatement(fnInnerBody);
       }
 
-      var headEnd, tailStart, innerHeaderStart, innerHeaderEnd;
+      var headEnd, tailStart, innerHeaderStart, innerHeaderEnd, hasUseStrict;
 
       if (fn && fn.body) {
         headEnd = fn.body.range[0] + 1;
         tailStart = fn.body.range[1] - 1;
         innerHeaderStart = headEnd;
         innerHeaderEnd = headStartWithRequire;
+
+        hasUseStrict = fnInnerBody.length && fnInnerBody[0].expression && fnInnerBody[0].expression.value === 'use strict';
```

Comment:
As mentioned, never used esprima before. Is this the correct way to check if first statement in the body is `use strict;`?